packages/contracts/scripts/shared/constants/addresses.ts

```diff
@@ -43,7 +42,6 @@ export const DEPLOYMENT_OVERRIDES: Record<string, any> = {
         MasterChefAddress: "0x8fFCf9899738e4633A721904609ffCa0a2C44f3D",
         MetaPoolAddress: "0x20955cb69ae1515962177d164dfc9522feef567e",
         BondingAddress: "0x831e3674Abc73d7A3e9d8a9400AF2301c32cEF0C",
-        BondingV2Address: "0xdAC17F958D2ee523a2206206994597C13D831ec7", // FAKE ADDRESS, TO BE REPLACED AFTER V2 DEPLOYMENT
         UbiquityAlgorithmicDollarManagerAddress: "0x4DA97a8b831C345dBe6d16FF7432DF2b7b776d98",
         jarUSDCAddr: "0xEB801AB73E9A2A482aA48CaCA13B1954028F4c94",
         jarYCRVLUSDaddr: "0x4fFe73Cf2EEf5E8C8E0E10160bCe440a029166D2",
```

Comment:
My comment in my review referred to deleting this entire file. Are these actually in use?

Reply:
> My comment in my review referred to deleting this entire file. Are these actually in use?

oh my bad! I removed it but didn't push.

Reply:
confirmed again. I removed it in https://github.com/ubiquity/ubiquity-dollar/pull/261/commits/399a6457f309087721e3b88689ac8bd4fb3af2ed